scripts/zones/Western_Adoulin/npcs/Auction_Counter.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- Area: Western Adoulin
+--  NPC: Auction Counter
+-----------------------------------
+require("scripts/quests/tutorial")
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    xi.tutorial.onAuctionTrigger(player)
```

Comment:
I don't think this auction counter can be used to complete the tutorial. So this line is unneeded